lib/screens/authenticate/sign_in.dart

```diff
@@ -22,81 +24,68 @@ class _SignInState extends State<SignIn> {
-                    TextFormField(
-                      decoration:
-                          textInputDecoration.copyWith(hintText: 'email'),
-                      validator: (val) => val.isEmpty ? 'Enter an email' : null,
-                      onChanged: (val) {
-                        setState(() => email = val);
-                      },
-                    ),
-                    SizedBox(height: 20.0),
-                    TextFormField(
-                      obscureText: true,
-                      decoration:
-                          textInputDecoration.copyWith(hintText: 'password'),
-                      validator: (val) => val.length < 6
-                          ? 'Enter a password 6+ chars long'
-                          : null,
-                      onChanged: (val) {
-                        setState(() => password = val);
-                      },
-                    ),
-                    SizedBox(height: 20.0),
-                    RaisedButton(
-                        color: Colors.pink[400],
-                        child: Text(
-                          'Sign In',
-                          style: TextStyle(color: Colors.white),
-                        ),
-                        onPressed: () async {
-                          if (_formKey.currentState.validate()) {
-                            setState(() => loading = true);
-                            dynamic result =
-                                await _auth.signIn(email, password);
-                            if (result == null) {
-                              setState(() {
-                                loading = false;
-                                error =
-                                    'Could not sign in with those credentials';
-                              });
-                            }
-                          }
-                        }),
-                    SizedBox(height: 12.0),
+        : new AuthLayout(
+            form: Form(
+              key: _formKey,
+              child: Column(
+                children: <Widget>[
+                  if (error != '')
                     Text(
                       error,
-                      style: TextStyle(color: Colors.red, fontSize: 14.0),
+                      style: TextStyle(
+                        color: Colors.red,
+                        fontSize: 14.0,
+                      ),
                     ),
-                  ],
-                ),
+                  SizedBox(
+                    height: 15,
+                  ),
+                  CustomTextFormField(
+                    placeholder: 'Email',
+                    validator: (val) => val.isEmpty ? 'Enter an email' : null,
+                    onChanged: (val) {
+                      setState(() => email = val);
+                    },
+                  ),
+                  SizedBox(
+                    height: 30,
+                  ),
+                  CustomTextFormField(
```

Comment:
Password field disappears when the keyboard is open.

![image](https://user-images.githubusercontent.com/8573325/102121367-f267dc80-3e54-11eb-8c34-9683b50d2849.png)
![image](https://user-images.githubusercontent.com/8573325/102121432-08759d00-3e55-11eb-8f1a-ed89b03372f8.png)

Reply:
Also this
![image](https://user-images.githubusercontent.com/8573325/102121507-204d2100-3e55-11eb-97f7-54fb42861802.png)

Reply:
Also, the warning is still there even if I enter an email.

![image](https://user-images.githubusercontent.com/8573325/102121565-378c0e80-3e55-11eb-8962-2128962f0e31.png)

Reply:
done